src/ostorlab/runtimes/local/models/alembic/env.py

```diff
@@ -0,0 +1,89 @@
+"""Alembic script that runs whenever the alembic migration tool is invoked.
+Its main responsibility is to invoke the migration engine."""
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+from alembic import context
+
+from ostorlab.runtimes.local.models.models import Base
+from ostorlab.runtimes.local.models import models as local_models
+from ostorlab.runtimes.local.models.models import Vulnerability # pylint: disable=W0611
+from ostorlab.runtimes.local.models.models import Scan # pylint: disable=W0611
+from ostorlab.runtimes.local.models.models import ScanStatus # pylint: disable=W0611
+
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = Base.metadata
+
+# other values from the config, defined by the needs of env.py,
+# can be acquired:
+# my_important_option = config.get_main_option("my_important_option")
+# ... etc.
+
+
+def run_migrations_offline() -> None:
+    """Run migrations in 'offline' mode.
+
+    This configures the context with just a URL
+    and not an Engine, though an Engine is acceptable
+    here as well.  By skipping the Engine creation
+    we don't even need a DBAPI to be available.
+
+    Calls to context.execute() here emit the given string to the
+    script output.
+
+    """
+    url = config.get_main_option('sqlalchemy.url')
+    context.configure(
+        url=url,
+        target_metadata=target_metadata,
+        literal_binds=True,
+        dialect_opts={'paramstyle': 'named'},
+        compare_type=True
+    )
+
+    with context.begin_transaction():
+        context.run_migrations()
+
+
+def run_migrations_online() -> None:
+    """Run migrations in 'online' mode.
+
+    In this scenario we need to create an Engine
+    and associate a connection with the context.
+
+    """
+    connectable = engine_from_config(
+        config.get_section(config.config_ini_section),
+        prefix='sqlalchemy.',
+        poolclass=pool.NullPool,
+    )
+
+    with connectable.connect() as connection:
+        context.configure(
+            connection=connection, target_metadata=target_metadata
+        )
+
+        with context.begin_transaction():
+            context.run_migrations()
+
+
+config.set_main_option('sqlalchemy.url', local_models.ENGINE_URL)
+
+if context.is_offline_mode():
+    run_migrations_offline()
+else:
+    run_migrations_online()
```

Comment:
Please use a __main__ if this script is to be invoked separately. I am not clear on what runs here and when.